### DISCUSSION TITLE
움직이는 오브젝트 위에서 아바타가 뚫리고 못따라가는 현상

### DISCUSSION THREAD

**Discussion by 3ANAKO** [naverz/zepeto-studio-kor#1178] [5. Q&A]
안녕하세요 

오브젝트는 애니메이션으로 위 아래로 움직입니다.

근데 여기에 아바타가 올라가서 가만히 있으면 괜찮은데 아바타를 움직이면 오브젝트를 뚫고 바닥으로 추락합니다. 

또한 오브젝트가 좌우로 움직일경우 아바타가 오브젝트 위에 있을때 오브젝트를 아바타가 따라가지않고 그 자리에 있다가 떨어집니다.

해결방안이 있을까요? 

![뚫림](https://user-images.githubusercontent.com/109119009/184785656-0b5e3200-f9d6-4172-9ab7-708a62c1bb98.png)

**Comment by zlmrzlo** [comment]
https://github.com/naverz/zepeto-studio-kor/discussions/902
위의  링크 참고하시면 될 것 같아요.
제페토에서 제공하는 기본 멀티플레이 예제인데 이걸로 공부하시면 도움될 것 같아요.
